src/components/Settings/RecorderSettings.tsx

```diff
@@ -60,6 +74,21 @@ export const RecorderSettings = () => {
         hint="Google Chrome needs to be installed on your machine for the recording functionality to work"
         disabled={recorder.detectBrowserPath}
       />
+
+      {!recorder.detectBrowserPath &&
+        recorder.browserPath !== '' &&
+        !isValidPath(recorder.browserPath?.toLocaleLowerCase()) && (
+          <Callout.Root color="amber">
+            <Callout.Icon>
+              <ExclamationTriangleIcon />
+            </Callout.Icon>
+
+            <Callout.Text>
+              The selected executable doesn&apos;t appear to be compatible.
+              Please select the correct executable for Chrome or Chromium.
```

Comment:
The hint only mentions Google Chrome. If we release this without the Chromium PR, it might be confusing

Reply:
I can make that change in the Chromium support PR once we fully validated it's working for all plaforms https://github.com/grafana/k6-studio/pull/535